src/csharp/Microsoft.Spark/Sql/Row.cs

```diff
@@ -36,6 +34,24 @@ internal Row(object[] values, StructType schema)
             Convert();
         }
 
+        /// <summary>
+        /// Constructor for the schema-less Row class used for chained UDFs.
+        /// </summary>
+        /// <param name="genericRow">GenericRow to convert from</param>      
```

Comment:
there is no conversion happening here.

Reply:
This constructor is for converting `GenericRow` to schema-less `Row`, right?

Reply:
well, it's just setting the given value to its member (not "convert"ing it)

Reply:
Got it, make sense.